torch/onnx/symbolic_opset11.py

```diff
@@ -11,7 +11,7 @@
 # This file exports ONNX ops for opset 11
 
 black_listed_operators = [
-    "eq", "ne", "scatter", "clamp", "clamp_min", "clamp_max", "sort", "topk", "hardtanh"
```

Comment:
unrelated change?

Reply:
Thi is not really unrelated. I realized that we already do support the updated version on "eq" and "ne" in opset 11 (The only update was support for more input types). So decided to remove them from this list.

Reply:
I meant unrelated to baddbmm. But I am ok with this, I was just checking.

Reply:
I would suggest to have a separate PR for this change.

Reply:
Sure